tracing.go

```diff
@@ -870,6 +870,22 @@ func WithSpanSampled(sampled Sampled) SpanOption {
 	}
 }
 
+func GetTraceHeader(s *Scope) string {
+	if s.span != nil {
+		return s.span.ToSentryTrace()
+	}
+
+	return fmt.Sprintf("%s-%s", s.propagationContext.TraceID, s.propagationContext.SpanID)
```

Comment:
@cleptric This may be wrong?

Reply:
wdym? The lack of a sample flag at the end?

Reply:
Yes. Is that mandatory?

Reply:
Nope, omitting it signals the upstream sdk to make its own sampling decision. In case of TwP, this is the desired behavior.
https://develop.sentry.dev/sdk/performance/#header-sentry-trace